app/controllers/FeedbackFormsController.scala

```diff
@@ -140,6 +143,22 @@ class FeedbackFormsController @Inject()(val messagesApi: MessagesApi,
     }
   }
 
+  def getFeedbackFormPreview: Action[JsValue] = AdminAction.async(parse.json) { implicit request =>
```

Comment:
Convert this to an action which accepts feedback form id as a part of URI as a GET request.